frontend/src/pages/ActivityReport/testHelpers.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
```

Comment:
This file isn't in _tests_ because if it was it would match our test regex, which I didn't want to mess with. The eslint warning says that its all right these packages are in devDependencies and not dependencies in package.json.

The stuff has been pulled out so I could write a smaller test file and not get weird run off from all kinds of other tests.